Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/TrackSelector.hpp

```diff
@@ -19,14 +19,14 @@ namespace ActsExamples {
 class TrackSelector final : public BareAlgorithm {
  public:
   struct Config {
-    /// Input track parameters collection.
+    /// Optional. Input track parameters collection.
```

Comment:
maybe add in the comment that parameters and trajectories are mutually exclusive (same below I guess)